ci/arm64-rpi/packer.build.arm64-rpi.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+if [ $# -gt 0 ]; then
+  pack=$1
+else
+  pack=lean
+fi
+
+if [ $# -gt 1 ]; then
+  github_user=$2
+else
+  github_user=rootzoll
+fi
+
+if [ $# -gt 2 ]; then
+  branch=$3
+else
+  branch=dev
+fi
+
+# Build the image in docker
+echo -e "\nBuild Packer image..."
+# from https://hub.docker.com/r/mkaczanowski/packer-builder-arm/tags
+docker run --rm --privileged -v /dev:/dev -v ${PWD}:/build \
+ mkaczanowski/packer-builder-arm:1.0.4@sha256:df09a8e249a292f10ca9b8cfd73420f5b987b6ac337d4ef28b6f4a8e61118822 \
```

Comment:
1.0.5 has the qemu integration fixed https://github.com/mkaczanowski/packer-builder-arm/releases/tag/v1.0.5 (does not require the `setup-qemu-action` to run before and also works locally)
```suggestion
 mkaczanowski/packer-builder-arm:1.0.5@sha256:a6371a3230b94c308e9a8cd55da1ae0d23bd670b7e558b39683c24f489d03e53 \
```

Reply:
That is great, thank you for looking through this. Locally we actually prefer to not run in Docker so there is one less dependency, but noted that could run the image generation in Docker as well now. Could be a good option on a non-clean system - like an existing OS not started for specifically for compiling.